nsdb-http/src/main/scala/io/radicalbit/nsdb/web/ApiResources.scala

```diff
@@ -96,7 +96,7 @@ class ApiResources(val publisherActor: ActorRef,
 
   implicit val formats: DefaultFormats = DefaultFormats
 
-  implicit val timeout: Timeout = Timeout(5 seconds)
+  implicit val timeout: Timeout = Timeout(50 seconds)
```

Comment:
Does it make sense to put this value in conf file?

Reply:
yes. I'm going to address this